get-started/graph-database/graph-database.adoc

```diff
@@ -20,93 +20,89 @@ You should have a basic understanding of what a database is.
 
 toc::[]
 
-=== Overview
+== Overview
 
-Very simply, a graph database is a database designed to treat the relationships between data as a first-class citizen in the data model.  
+Very simply, a graph database is a database designed for storing and retrieving data and the relationships among it.
+Rather than prioritizing a rigidly-formatted structure to force on the data and navigating the structure to analyze sets of data, a graph database directly connects each piece of data to those it is related to, elevating the relationships in the data model to equal importance with the data itself.
 
 ++++
-<iframe width="640" height="360" src="https://www.youtube.com/embed/-dCeFEqDkUI" frameborder="0" allowfullscreen></iframe>
+<iframe width="680" height="425" src="https://www.youtube.com/embed/_D19h5s73Co" frameborder="0" allowfullscreen></iframe>
 ++++
 
 
-=== Why Graph Databases?
+== Why Graph Databases?
 
-We live in a connected world!  There are no isolated pieces of information, but rich, connected domains all around us.
+We live in a connected world!
+There are no isolated pieces of information, but rich, connected domains all around us.
 Only a database that natively embraces relationships is able to store, process, and query connections efficiently.
-While other databases compute relationships at query time through expensive JOIN operations, a graph database stores connections as first class citizens.
+While other databases compute relationships at query time through expensive JOIN operations, a graph database stores connections alongside the data in the model.
```

Comment:
Just curious-are you intentionally removing all "first class citizen" text?

Reply:
Ya me too.

Reply:
I found that comparison a little confusing when I first started. It makes perfect sense to me now (and is a perfect comparison), but I wanted to make the initial definition as simple and straightforward as possible. Also, I feel this wording might be even more confusing for various levels of English understanding.